docs/client/full-api/api/templates.md

```diff
@@ -85,6 +85,33 @@ initialization or clean-up on the object.
 Because your template has been rendered, you can use functions like
 [`this.findAll`](#template_findAll) which look at its DOM nodes.
 
+This can be a good place to apply any DOM manipulations you want, after the
```

Comment:
Yeah, it was a leftover from my previous iteration of this, where I had an autorun in side a rendered callback